src/features/settings/gestures/gesturesSlice.tsx

```diff
@@ -0,0 +1,154 @@
+import { PayloadAction, createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import {
+  SwipeAction,
+  SwipeActions,
+  db,
+  default_swipe_actions_post,
+  default_swipe_actions_comment,
+  default_swipe_actions_inbox,
+} from "../../../services/db";
+
+interface GestureState {
+  swipe: {
```

Comment:
Probably doesn't need to be nested in `swipe`?

Reply:
I was going to do that so other gesture stuff would not be nested (e.g. disableBackGesture, which I am working on adding).